website2/docs/compiling-linux.md

```diff
@@ -39,8 +39,8 @@ $ sudo apt-get upgrade -y
 ### Step 2 --- Install required libraries
 
 ```bash
-$ sudo apt-get install git build-essential automake cmake libtool-bin zip \
-  libunwind-setjmp0-dev zlib1g-dev unzip pkg-config python-setuptools -y
+$ sudo apt-get install git build-essential automake cmake libtool-bin zip ant \
+  libunwind-setjmp0-dev zlib1g-dev unzip pkg-config python-setuptools libcppunit-dev -y
```

Comment:
I think this needs to be `python3-setuptools`. Slipped through the cracks from our Python 3 upgrade.